### PR TITLE
XW-1807 | remove code for old GlobalFooter

### PR DESCRIPTION
**Pull request by hakubo**
and enable DS Global Navigation by default

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 22 issues
- MAJOR 5 major
- MINOR 17 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiaHomePageHelper.class.php#L15](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L15): Class "WikiaHomePageHelper" has 52 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1448)
2. MAJOR [WikiaHomePageHelper.class.php#L177](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L177): Add curly braces around the nested statement(s). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)
3. MAJOR [WikiaHomePageHelper.class.php#L342](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L342): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)
4. MAJOR [WikiaHomePageHelper.class.php#L565](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L565): Remove this commented out code. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)
5. MAJOR [WikiaHomePageHelper.class.php#L574](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L574): Remove this commented out code. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)
6. MINOR [WikiaHomePageHelper.class.php#L177](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L177): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS122)
7. MINOR [WikiaHomePageHelper.class.php#L230](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L230): Define a constant instead of duplicating this literal "articles" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
8. MINOR [WikiaHomePageHelper.class.php#L231](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L231): Define a constant instead of duplicating this literal "pages" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
9. MINOR [WikiaHomePageHelper.class.php#L232](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L232): Define a constant instead of duplicating this literal "images" 13 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
10. MINOR [WikiaHomePageHelper.class.php#L233](https://github.com/Wikia/app/blob/442a80e9431c4c457ebc1cc48122705388b44d6b/extensions/wikia/CityVisualization/helpers/WikiaHomePageHelper.class.php#L233): Define a constant instead of duplicating this literal "users" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)

**Review by tomasznapieralski — APPROVED** [review]

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 1]
```diff
@@ -17,10 +17,6 @@ $color-borders: #CECECE;
 	}
 }
 
-.CorporateFooter {
-	display: none;
-}
-
 .WikiaFooter {
```
Comment (rogatty) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss:20]: This container is now empty so maybe it doesn't make sense to keep this class here?

File: extensions/wikia/GlobalFooter/crowdin.conf [hunk 2]
```diff
@@ -1,4 +0,0 @@
-[crowdin.project]
-projectid = "ext-GlobalFooter"
```
Comment (rogatty) [inline extensions/wikia/GlobalFooter/crowdin.conf:2]: Do we plan to remove this project from Crowdin?

**Review by rogatty — APPROVED** [review]

**Review by Warkot — APPROVED** [review]

**Review by tomasznapieralski — APPROVED** [review]
Tested

**Comment by macbre** [comment]
Please remove an `include` from config as well

```
Linting /usr/wikia/source/config/CommonExtensions.php...
2532: extensions/wikia/CorporateFooter/CorporateFooter.setup.php can not be found
        include("$IP/extensions/wikia/CorporateFooter/CorporateFooter.setup.php");
```

**Pull request merged**